### ISSUE TITLE
The factors of 4 loss

### ISSUE THREAD

**Issue by andyzj2621** [TUI-NICR/ESANet#14]
I'm studying your network,it is very good and interesting.I spent much time in looking for 4 factors of 4 training loss.
In other words,i can not find a,b,c,d,  in   All-training-loss  =a * loss_from_the_last_output +b * loss_from_first_decodermodule
+c * loss_from_second_decodermodule +d * loss_from_third_decodermodule .what are them and how can i change them.I'd appreciate it if you could give me a cue.Thank you very much.

**Comment by mona0809** [comment]
The losses are computed here: https://github.com/TUI-NICR/ESANet/blob/main/src/utils.py#L34
The function returns a list with the 4 losses. These 4 losses are simply summed up here: https://github.com/TUI-NICR/ESANet/blob/main/train.py#L306. That means all 4 factors equal 1. Instead of summing them up, you could define weighting factors yourself.

The order of the values in the list is the following:
- loss for the full size output
- loss for 1/8 size output
- loss for 1/16 size output
- loss for 1/32 size output

Let us know if you have any further questions.

**Comment by andyzj2621** [comment]
Thank for you help ,you are so kind

**Comment by andyzj2621** [comment]
Thank for you help ,you are so kind.:smile: @mona0809
